### PR TITLE
Create a random character on update to 2.0

### PR DESCRIPTION
**Pull request by convolu**
Generate an initial character for users that update

### CODE REVIEW

**Comment by convolu** [comment]
@pazdera @alex5imon Could you please review?

File: kano_updater/scenarios.py [hunk 1]
```diff
@@ -278,6 +279,11 @@ def beta_132_to_beta_133(self):
...
         pass
-    
+
     def beta_134_to_beta_200(self):
-        pass
+        if is_installed('kano-character-cli'):
+            cmd = 'kano-character-cli -c "Judoka_Base" "Hair_Black" "Skin_Orange" -r -s'
+            run_cmd_log(cmd)
+        else:
+            logger.warn(
```
Comment (alex5imon) [inline kano_updater/scenarios.py]: Should this just be a warning? Shouldn't we try to fix it by installing it (and any other dependencies)?

Comment (convolu) [inline kano_updater/scenarios.py]: This should happen in the Post Update step. If `kano-character-cli` is missing, it probably means that there is something wrong with the update. Should I bump this to error?

Comment (alex5imon) [inline kano_updater/scenarios.py]: Definitely to error but you might also want to fix the problem

Comment (pazdera) [inline kano_updater/scenarios.py]: @alex5imon I think that the role of the check is that if the update fails really badly, this won't make it worse. If you don't have the `kano-profile` installed that usually means that your install is pretty badly damaged.

Comment (convolu) [inline kano_updater/scenarios.py]: Updated in https://github.com/KanoComputing/kano-updater/commit/a8117e1da1d87407c3ef99f0d857b449d9a6ad3e

**Comment by convolu** [comment]
Are these changes ok, may I merge?

**Comment by pazdera** [comment]
@convolu Looks good to me!

**Comment by alex5imon** [comment]
Please merge :+1:

**Comment by convolu** [comment]
Thanks, merging

**Pull request merged**